ivy_tests/test_ivy/test_frontends/test_paddle/test_manipulation.py

```diff
@@ -439,37 +439,7 @@ def test_paddle_gather(
 # gather_nd
 @handle_frontend_test(
     fn_tree="paddle.gather_nd",
-    dtype_x_index=helpers.array_indices_axis(
```

Comment:
gather_nd is also accessible via `paddle.gather_nd`. No need to remove the original function or this test.

Reply:
Same here. Should I put it back and push to remote branch? Thanks.

Reply:
Yes, you can.